.docker/000-default.conf

```
@@ -0,0 +1,22 @@
+<VirtualHost *:8001>
+    ServerAdmin webmaster@localhost
+    DocumentRoot /opt/kimai/public
+
+    PassEnv DATABASE_PREFIX
+    PassEnv MAILER_FROM
+    PassEnv APP_ENV
+    PassEnv APP_SECRET
+    PassEnv DATABASE_URL
+    PassEnv MAILER_URL
+    PassEnv TRUSTED_PROXIES
+    PassEnv TRUSTED_HOSTS
```

```suggestion
    PassEnv MAILER_FROM
    PassEnv APP_ENV
    PassEnv APP_SECRET
    PassEnv DATABASE_URL
    PassEnv MAILER_URL
    PassEnv TRUSTED_PROXIES
    PassEnv TRUSTED_HOSTS
    PassEnv CORS_ALLOW_ORIGIN
```

removed DATABASE_PREFIX as it does not exist anymore and added CORS_ALLOW_ORIGIN